### ISSUE TITLE
README in regular english

### ISSUE THREAD

**Issue by selfawaresoup** [yargs/yargs#46]
The "pirate" style README is unnecessarily hard to read and understand. Writing it in plain english would make it more accessible and comprehensible.

Not only will it enable most people to get help faster and more easily but it will prevent serious understanding issues for non-native english speakers.

**Comment by adsteel** [comment]
What happened to this "much easier to read" readme? As a native English speaker and fan of International Talk Like A Pirate Day, I find it very difficult to wade through the current readme language.

**Comment by maxrimue** [comment]
@adsteel if I remember correctly we still have a lot planned for docs, as we eventually wanted to split up documentation into specific parts and files due to yargs' growing list of features (and other tiny things to make docs easier to read altogether) - I'll open a new tracking issue for this

**Comment by adsteel** [comment]
@maxrimue Sounds good! If standardizing the language ends up in that plan, I'd appreciate it. Thanks for all your work!

**Comment by maxrimue** [comment]
@adsteel you're welcome! See #809
